packages/react-bootstrap-table2/src/header-cell.js

```diff
@@ -68,7 +69,7 @@ const HeaderCell = (props) => {
     cellAttrs.className = cs(cellAttrs.className, 'sortable');
 
     if (sorting) {
-      sortSymbol = <SortCaret order={ sortOrder } />;
+      sortSymbol = sortCaret ? sortCaret(sortOrder) : <SortCaret order={ sortOrder } />;
```

Comment:
I think `sortCaret` function can pass one more argument: `column`, because some user will custom the caret by different column. `column` should be first argument then is the `sortOrder`

Reply:
I disagree on the use case. That would need more when someone (you, me, etc...) implements multi sort capability.

Reply:
what's your concern for passing column object? @mcystems

Reply:
@mcystems any idea about above comment?